app/views/WhatYouOwe.scala.html

```diff
@@ -44,6 +44,8 @@
                 id=Some("sa-note-migrated"))
             @p(content = Html(messages("whatYouOwe.outstanding-charges-note")), id=Some("outstanding-charges-note-migrated"))
         </div>
+    } else {
+        @p(content = Html(""), id = Some("no-payments-due-note"))
```

Comment:
don't do this, adjust the margin in the styling with something along the line of [govuk-!-padding-bottom-9]